Moodle_2servers/Moodle_2servers.yaml

```diff
@@ -0,0 +1,326 @@
+#
+# Infrastructure template for Moodle on two instances plus all network infrastructure.
+#
+# DNS pre-configured for the Porirua (nz-por-1) region of the Catalyst Cloud.
+#
+
+heat_template_version: 2013-05-23
+
+description: >
+  Infrastructure template for Moodle on two instances plus all network infrastructure.
+parameter_groups:
+- label: Infrastructure
+  description: Infrastructure configuration
+  parameters:
+  - key_name
+  - servers_flavor
+  - image
+  - public_net
+  - public_net_id
+  - private_net_name
+  - private_net_cidr
+  - private_net_gateway
+  - private_net_pool_start
+  - private_net_pool_end
+  - private_net_dns_servers
+  - router_name
+- label: Moodle
+  description: Moodle instance configuration
+  parameters:
+  - site_name
+  - environment
+  - app_type
+  - site_url
+  - moodle_version
+- label: Database
+  description: Database configuration
+  parameters:
+  - db_root_password
+
+parameters:
+
+# Infrastructure
+
+  key_name:
+    type: string
+    description: Name of an existing key pair to use for the server
+    constraints:
+      - custom_constraint: nova.keypair
+    default: cristiano
```

Comment:
Yep, thanks for spotting it. I thought I had removed it everywhere, but I was wrong